udata/core/dataset/constants.py

```diff
@@ -4,42 +4,62 @@
 
 #: Udata frequencies with their labels
 #:
-#: See: http://dublincore.org/groups/collections/frequency/
+#: See: https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/frequency
 UPDATE_FREQUENCIES = OrderedDict(
-    [  # Dublin core equivalent
-        ("unknown", _("Unknown")),  # N/A
-        ("punctual", _("Punctual")),  # N/A
-        ("continuous", _("Real time")),  # freq:continuous
-        ("hourly", _("Hourly")),  # N/A
-        ("fourTimesADay", _("Four times a day")),  # N/A
-        ("threeTimesADay", _("Three times a day")),  # N/A
-        ("semidaily", _("Semidaily")),  # N/A
-        ("daily", _("Daily")),  # freq:daily
-        ("fourTimesAWeek", _("Four times a week")),  # N/A
-        ("threeTimesAWeek", _("Three times a week")),  # freq:threeTimesAWeek
-        ("semiweekly", _("Semiweekly")),  # freq:semiweekly
-        ("weekly", _("Weekly")),  # freq:weekly
-        ("biweekly", _("Biweekly")),  # freq:bimonthly
-        ("threeTimesAMonth", _("Three times a month")),  # freq:threeTimesAMonth
-        ("semimonthly", _("Semimonthly")),  # freq:semimonthly
-        ("monthly", _("Monthly")),  # freq:monthly
-        ("bimonthly", _("Bimonthly")),  # freq:bimonthly
-        ("quarterly", _("Quarterly")),  # freq:quarterly
-        ("threeTimesAYear", _("Three times a year")),  # freq:threeTimesAYear
-        ("semiannual", _("Biannual")),  # freq:semiannual
-        ("annual", _("Annual")),  # freq:annual
-        ("biennial", _("Biennial")),  # freq:biennial
-        ("triennial", _("Triennial")),  # freq:triennial
-        ("quinquennial", _("Quinquennial")),  # N/A
-        ("irregular", _("Irregular")),  # freq:irregular
+    [
+        # FIXME: Add ALL EU terms (from 30yrs to 1mn)?
+        # FIXME: Uppercase to match EU vocab?
+        # FIXME: Drop end-of-line annotations
+        ("unknown", _("Unknown")),  # =
+        ("other", _("Other")),  # N/A
+        ("never", _("Never")),  # N/A
+        ("not_planned", _("Not planned")),  # N/A
+        ("as_needed", _("As needed")),  # punctual
```

Comment:
Breaking change : API users will need to update to the new vocabulary. Is that OK?

An alternative is to keep the existing terms and only use EU vocab for new terms. I'm not fan of that option since we'll effectively create a franken-vocab for the udata API.

Reply:
@maudetes This is the most important point to decide. WDYT?

Reply:
Let's keep existing terms then. I agree it's a lot to break.
We can always control the mapping to be full EU when exporting to specific standards, if necessary.

Reply:
I'm tempted to change things like `threeTimesAWeek` to `weekly_3` so we have a more consistent vocabulary overall: `[weekly, weekly_2, weekly_3, weekly_5]` seems better than `[weekly, semiweekly, threeTimesAWeek, weekly_5]`.